ch07.adoc

```diff
@@ -544,3 +544,168 @@ data:  // time coordinates translated to date/time format
                      "2000-8-1 6:00:00", "2000-9-1 6:00:00" ;
 ----
 ====
+
+[[geometries, Section 7.5, "Geometries"]]
+=== Geometries
+
+For many geospatial applications, data values are associated with a geometry, which is a spatial representation of a real-world feature, for instance a time-series of areal average precipitation over a watershed. 
+Polygonal cells with an arbitrary number of vertices can be described using <<cell-boundaries>>, but in that case every cell must have the same number of vertices and must be single polygon rings. 
+In contrast, each geometry may have a different number of nodes, the geometries may be lines (as alternatives to points and polygons), and they may be __multipart__ i.e include several disjoint parts. 
+While line and point geometries don't describe an interval along a dimension as the traditional cell bounds described above do, they do describe the extent of a geometry or real-world feature so are included in this section. 
```

Comment:
@dblodgett-usgs I think this is looking good! Thanks. Some minor comments from my latest read

* Example 7.15: Needs a data variable to complete it
* It would be handy to have a table defining the different types: line, multiline, point, multipolygon, etc. (Should this be in-line or in an appendix?)
* Appendix A needs updating for the new attributes (or has this been done elsewhere, like the the grid mapping variables?)

All the best, David